devtools/dump_devinfo.py

```diff
@@ -35,6 +34,8 @@ def scrub(res):
         "longitude_i",
         "latitude",
         "longitude",
+        "device_owner_hash",
+        "device_id_hash",
```

Comment:
Would you also mind adding the other keys in the "tapo" payloads to this list of keys? I'm seeing at least the following in the info reply for my new test bulb:
* `device_id`
* `hw_id`
* `ip` 
* `ssid`
* `fw_id` (maybe, unsure what this is?)
* `oem_id` (maybe, unsure what this is)

I'm wondering if `time_diff` and `region` should also be scrubbed, but I think those are not that unique and could be useful for tests, too.

Reply:
I plan to add them but thought it better to add to the TAPO PR. As it stands this PR works with the devices currently supported by the master branch.

Reply:
Ok I added them including fw_id and oem_id to be safe